docs/guide/dubboForNode/ImplementingServices.md

```diff
@@ -1 +1,234 @@
 # ImplementingServices
+
+Dubbo handles HTTP routes and most plumbing for you, but implementing the actual business logic is still up to you.
+
+You always register your implementation on the `DubboRouter`. We recommend to create a file `connect.ts` with a registration function in your project:
```

Comment:
You always register your implementation on the `DubboRouter`. We recommend to create a file `dubbo.ts` with a registration function in your project: